### PR TITLE
feat: added text overrides on CheckoutActions

### PR DESCRIPTION
**Pull request by HarisSpahijaPon**
Part of: #409 
Impact: **minor**  
Type: **feature**

## Component
CheckoutActions now has overrides when i18next will be implemented (see #409)

## Breaking changes
No breaking chances, all added fields are optional

## Testing
1. Add a new prop `cancelButtonText` to `<CheckoutActions />` 
2. Add the value `"test"` to `cancelButtonText`
3. Button should display "test" when test is given as a value to the prop. When the prop `cancelButtonText` is not added, it should default to the values given in default props

## Added props:
```
cancelButtonText: PropTypes.string,
isNotSavingButtonText: PropTypes.string,
isSavingButtonText: PropTypes.string,
saveButtonText: PropTypes.string,
```

### CODE REVIEW

**Comment by rc-publisher** [comment]
🚀 Preview deployed

Built with commit 08bd12ea331b11c2e4cd006857fe53b9f98f1ebc

https://deploy-preview-453--stoic-hodgkin-c0179e.netlify.com

**Comment by rc-publisher** [comment]
🚀 Preview deployed

Built with commit 7e45ee860d6bb21b9c4ce4978e8920344c27755c

https://deploy-preview-453--stoic-hodgkin-c0179e.netlify.com

File: package/src/components/CheckoutActions/v1/CheckoutActions.js [hunk 1]
```diff
@@ -246,21 +267,21 @@ class CheckoutActions extends Component {
...
+          {saveButtonText}
         </Button>
       </React.Fragment>
     );
 
     const placeOrderButton = (
       <PlaceOrderButtonContainer>
         <Button onClick={() => this.actionSubmit(action.id)} actionType="important" isWaiting={isSaving} isFullWidth>
-          {isSaving ? "Placing your order..." : "Place your order"}
+          {isSaving ? `${isSavingButtonText}` : `${isNotSavingButtonText}`}
```
Comment (aldeed) [inline package/src/components/CheckoutActions/v1/CheckoutActions.js]: Template literals aren't necessary here. Change to:

```js
{isSaving ? isSavingButtonText : isNotSavingButtonText}
```

**Pull request merged**